randovania/games/am2r/json_data/The Tower.txt

```diff
@@ -248,7 +260,7 @@ Extra - map_name: rm_a4h04
   > Dock to Exterior Zeta Nest West Access
       Trivial
   > Door to Tower Activation Station
-      Mid-Air Morph (Advanced) and Movement (Intermediate)
+      Morph Ball and Mid-Air Morph (Advanced) and Movement (Intermediate)
```

Comment:
i think the mid air morph can be removed here. pretty sure the glide can also be done by just shooting (i.e. movement)?

Reply:
I can do plasma tunnel to gamma with the beam cancels, but gamma to tower activation seems too tight to do without the morph glide (skill issue?)